packages/scripts/config/baseModuleRollupConfig.js

```diff
@@ -0,0 +1,29 @@
+const baseRollupConfig = require('./baseRollupConfig');
+const angularJsTemplateLoader = require('../helpers/rollup-plugin-angularjs-template-loader');
+
+module.exports = {
+  ...baseRollupConfig,
+  output: [{ dir: 'dist', format: 'es', sourcemap: true }],
+  plugins: [...baseRollupConfig.plugins, angularJsTemplateLoader({ sourceMap: true })],
+  // NOTE: The following list of external dependencies will be updated as more spinnaker modules are migrated to
+  // use rollup.
+  external: [
```

Comment:
I think for modules we should make everything external as @ranihorev suggests.  Let webpack figure it out when it bundles everything.

Reply:
Cool, I'm going to use `rollup-plugin-auto-external` to automatically handle it.